runtime/oti/bcverify_api.h

```diff
@@ -176,7 +176,7 @@ j9bcv_checkClassLoadingConstraintsForSignature (J9VMThread* vmThread, J9ClassLoa
 * @return UDATA
  */
 UDATA
-j9bcv_checkClassLoadingConstraintForName (J9VMThread* vmThread, J9ClassLoader* loader1, J9ClassLoader* loader2, U_8* name1, U_8* name2, UDATA length, UDATA copyUTFs);
+j9bcv_checkClassLoadingConstraintForName (J9VMThread *vmThread, J9ClassLoader *loader1, J9ClassLoader *loader2, U_8 *name1, U_8 *name2, UDATA length, UDATA copyUTF1, UDATA copyUTF2);
```

Comment:
`copyUTF1 & copyUTF2` isn't clear, I would prefer `copyName1/2` similar to what you've done for Signature function.